imports/plugins/core/orders/client/graphql/index.js

```diff
@@ -28,3 +31,28 @@ export const captureOrderPayments = async ({ orderId, paymentIds, shopId }) => {
     }
   });
 };
+
+export const updateOrderFulfillmentGroup = async ({
+  orderId,
+  orderFulfillmentGroupId,
+  status,
+  tracking
```

Comment:
See that you noted trackingUrl is not updated here yet. Should we go ahead and add it already to keep this in sync with what the other mutation can do?

Reply:
I'd rather wait until we actually add a field in UI for it and need it here.